webrtc.html

```diff
@@ -8324,7 +8327,11 @@ <h3><dfn>RTCIceTransport</dfn> Interface</h3>
       <p>When the <a>ICE Agent</a> indicates that a new ICE candidate is
       available for an <code><a>RTCIceTransport</a></code>, either by taking one
       from the <a data-lt="ICE candidate pool size">ICE candidate pool</a> or
-      gathering it from scratch, the user agent MUST queue a task that runs the
+      gathering it from scratch, the user agent MUST wait until either the
```

Comment:
the user agent MUST put the candidate into a processing queue [[queue]] for that transport, and do the check below.

When the currentLocalDescription or PendingLocalDescription indicate that the ICE generation is teh one for which candidates in the processing queue are intended, it MUST queue a task that...

Reply:
Also add to SLD: "If a new SLD call is initiated when there are candidates in [[queue]], clear the queue."